app/migrations/versions/917051c561c8_.py

```diff
@@ -0,0 +1,30 @@
+"""Make custom attributes plain old ordinary strings
+
+Revision ID: 917051c561c8
+Revises: 2c3f97929457
+Create Date: 2020-09-03 16:05:10.156853
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '917051c561c8'
+down_revision = '2c3f97929457'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('custom_attribute_user_storage', 'value',
```

Comment:
Curious why this isn't better as JSON (or even JSONB) - doesn't that allow us to do things like query all values where it's `True` as a boolean, rather than `True` as a string?

Reply:
https://sempo.slack.com/archives/CLH7NENGJ/p1594320862001000